features/postmessage.yml

```diff
@@ -1,16 +1,27 @@
-name: PostMessage
-description: "The `postMessage()` method safely enables cross-origin communication for windows and workers, including popups and iframes."
+name: postMessage
+description: "The `postMessage()` global method sends cross-origin messages to windows and workers, including popups and iframes. Also known as cross-document messaging."
 spec: https://html.spec.whatwg.org/multipage/web-messaging.html#web-messaging
 caniuse: x-doc-messaging
 group: messaging
 # Anchor baseline on window support.
 status:
   compute_from: api.Window.postMessage
 compat_features:
-  # Already generated in Broadcast channel.
-  # Revisit once the "keys in impl or feature" question is answered.
-  #- api.BroadcastChannel.postMessage
+  # Other features, such as channel messaging and broadcast channels have
+  # `postMessage` methods too, but are not covered by this feature.
   - api.DedicatedWorkerGlobalScope.postMessage
-  - api.MessagePort.postMessage
   - api.Window.postMessage
   - api.Worker.postMessage
+
+  # Already generated in Broadcast channel.
+  # Revisit once the "keys in impl or feature" question is answered.
+  #- api.BroadcastChannel.message_event
```

Comment:
This comment is unclear, more so now that we have the context-setting comment above. I think we can drop the whole thing.

```suggestion
```

Reply:
yeh, already had done locally